README.md

```diff
@@ -43,10 +43,10 @@ If you want a simple guide to install Empirical Core, then you've come to the ri
 
 9. Run Redis with ```redis-server```
 
-11. Run npm installer with ```npm install && cd ./client && npm install```
```

Comment:
This changes looks good, as the step of cd'ing into the client folder and running `npm install` at that subsidiary level is addressed in the `postinstall` npm script.

Can I suggest you also make the same change to the documentation, in `README.md` at step 11 and delete step 12 as it would now be unnecessary?